ACID/acid_tbc.sql

```diff
@@ -21041,6 +21041,9 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 ('2296701','22967','0','0','100','1','1000','20000','20000','240000','11','30931','0','32','0','0','0','0','0','0','0','0','Scryer Cavalier - Cast Battle Shout'),
 ('2296702','22967','0','0','100','1','5000','15000','15000','25000','11','35871','1','32','0','0','0','0','0','0','0','0','Scryer Cavalier - Cast Spellbreaker'),
 ('2296703','22967','6','0','100','0','0','0','0','0','43','0','0','0','11','39783','0','0','0','0','0','0','Scryer Cavalier - Dismount and Cast Summon Eclipsion Hawkstrider on Death'),
+-- Overmine Flayer
+('2326401','23264','0','0','100','1','5000','20000','15000','20000','11','13443','1','0','0','0','0','0','0','0','0','0','Overmine Flayer - Cast Rend'),
+('2326402','23264','2','0','100','0','60','0','0','0','11','33896','0','0','54','-124','0','0','0','0','0','0','Overmine Flayer - Cast Desperate Defense and Text Emote at 60% HP'),
```

Comment:
Should be s.33810
Repeat timer 10-15s (below 60%hp only)
Keeps casting shield even if already enraged